app/bundles/LeadBundle/Assets/js/lead.js

```diff
@@ -244,6 +244,25 @@ Mautic.getLeadId = function() {
 }
 
 Mautic.leadlistOnLoad = function(container, response) {
+    const segmentCountElem = mQuery('a.col-count');
```

Comment:
This PR is not complete.
The class `.col-count`  in count href tag here https://github.com/mautic/mautic/blob/ac66bcc8dac581d8b2ed8094f7f1e09ea7763515/app/bundles/LeadBundle/Views/List/list.html.php#L136

If I add it, then ajax call return error  (some other method haven't cherry picked)

![image](https://user-images.githubusercontent.com/462477/152637277-1f81bbe1-6bcc-404c-a1b9-7f230438fbfa.png)

Reply:
@shinde-rahul are you able to look on my comment? This looks like great PR, we would like to include it to 4.2.1

Reply:
@shinde-rahul are you able to check the error?

Reply:
@kuzmany, some commits got added along the way. I have rebased this with 4.4 now.